build/rpm/etc/init.d/kcnd

```diff
@@ -178,37 +257,57 @@ CURRENTFILE=`basename "$0"`
 OPTIONS="$OPTIONS --autorestart.daemon.path $BASEDIR/$CURRENTFILE"
 set +f
 
-start() {
-        echo -n "Starting $prog: "
-        if [ ! -d ${LOG_DIR} ]; then
-                mkdir -p ${LOG_DIR}
-        fi
+start_node() {
+    echo -n "Starting $prog: "
+    if [ ! -d ${LOG_DIR} ]; then
+            mkdir -p ${LOG_DIR}
+    fi
```

Comment:
Do you know why rpm package doesn't check the existence of `DATA_DIR`? 
I know it has worked well without the checking logic now. I am just wondering.

Reply:
As far as I know, DATA_DIR defined in .conf file created when node was installed through the rpm package.
Therefore, I think there is no check logic whether the existence of DATA_DIR.